Fix vulnerabilites in dependencies

```
[ERROR] Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:3.2.0:audit (default-cli) on project virtual-schema-common-document-files: Detected 10 vulnerable components:
[ERROR]   com.squareup.okhttp:okhttp:jar:2.7.5:compile; https://ossindex.sonatype.org/component/pkg:maven/com.squareup.okhttp/okhttp@2.7.5?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2021-0341] CWE-295: Improper Certificate Validation (7.5); https://ossindex.sonatype.org/vulnerability/CVE-2021-0341?component-type=maven&component-name=com.squareup.okhttp%2Fokhttp&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [sonatype-2018-0035] CWE-20: Improper Input Validation (5.9); https://ossindex.sonatype.org/vulnerability/sonatype-2018-0035?component-type=maven&component-name=com.squareup.okhttp%2Fokhttp&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   io.netty:netty-common:jar:4.1.72.Final:test; https://ossindex.sonatype.org/component/pkg:maven/io.netty/netty-common@4.1.72.Final?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2022-24823] CWE-668: Exposure of Resource to Wrong Sphere (5.5); https://ossindex.sonatype.org/vulnerability/CVE-2022-24823?component-type=maven&component-name=io.netty%2Fnetty-common&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   org.apache.hadoop:hadoop-common:jar:3.3.1:compile; https://ossindex.sonatype.org/component/pkg:maven/org.apache.hadoop/hadoop-common@3.3.1?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2022-26612] CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') (9.8); https://ossindex.sonatype.org/vulnerability/CVE-2022-26612?component-type=maven&component-name=org.apache.hadoop%2Fhadoop-common&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   com.google.guava:guava:jar:31.0.1-jre:compile; https://ossindex.sonatype.org/component/pkg:maven/com.google.guava/guava@31.0.1-jre?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [sonatype-2020-0926] CWE-379: Creation of Temporary File in Directory with Incorrect Permissions (6.2); https://ossindex.sonatype.org/vulnerability/sonatype-2020-0926?component-type=maven&component-name=com.google.guava%2Fguava&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   io.netty:netty-handler:jar:4.1.72.Final:test; https://ossindex.sonatype.org/component/pkg:maven/io.netty/netty-handler@4.1.72.Final?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [sonatype-2020-0026] CWE-300: Channel Accessible by Non-Endpoint ('Man-in-the-Middle') (6.5); https://ossindex.sonatype.org/vulnerability/sonatype-2020-0026?component-type=maven&component-name=io.netty%2Fnetty-handler&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   org.apache.xmlrpc:xmlrpc-common:jar:3.1.3:test; https://ossindex.sonatype.org/component/pkg:maven/org.apache.xmlrpc/xmlrpc-common@3.1.3?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2016-5003] CWE-502: Deserialization of Untrusted Data (9.8); https://ossindex.sonatype.org/vulnerability/CVE-2016-5003?component-type=maven&component-name=org.apache.xmlrpc%2Fxmlrpc-common&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2016-5002] CWE-611: Improper Restriction of XML External Entity Reference ('XXE') (7.8); https://ossindex.sonatype.org/vulnerability/CVE-2016-5002?component-type=maven&component-name=org.apache.xmlrpc%2Fxmlrpc-common&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   com.google.protobuf:protobuf-java:jar:2.5.0:compile; https://ossindex.sonatype.org/component/pkg:maven/com.google.protobuf/protobuf-java@2.5.0?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2021-22569] CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion') (5.5); https://ossindex.sonatype.org/vulnerability/CVE-2021-22569?component-type=maven&component-name=com.google.protobuf%2Fprotobuf-java&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   commons-codec:commons-codec:jar:1.11:compile; https://ossindex.sonatype.org/component/pkg:maven/commons-codec/commons-codec@1.11?utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1
[ERROR]     * [sonatype-2012-0050] CWE-20: Improper Input Validation (5.3); https://ossindex.sonatype.org/vulnerability/sonatype-2012-0050?component-type=maven&component-name=commons-codec%2Fcommons-codec&utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1
[ERROR]   org.apache.xmlrpc:xmlrpc-client:jar:3.1.3:test; https://ossindex.sonatype.org/component/pkg:maven/org.apache.xmlrpc/xmlrpc-client@3.1.3?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [CVE-2016-5004] CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion') (6.5); https://ossindex.sonatype.org/vulnerability/CVE-2016-5004?component-type=maven&component-name=org.apache.xmlrpc%2Fxmlrpc-client&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]   com.google.code.gson:gson:jar:2.2.4:compile; https://ossindex.sonatype.org/component/pkg:maven/com.google.code.gson/gson@2.2.4?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
[ERROR]     * [sonatype-2021-1694] CWE-502: Deserialization of Untrusted Data (7.5); https://ossindex.sonatype.org/vulnerability/sonatype-2021-1694?component-type=maven&component-name=com.google.code.gson%2Fgson&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
```

--- a/.github/workflows/broken_links_checker.yml
+++ b/.github/workflows/broken_links_checker.yml
@@ -2,7 +2,7 @@ name: Broken Links Checker
 
 on:
   schedule:
-    - cron: "0 5 * * *"
+    - cron: "0 5 * * 0"
   push:
     branches:
       - main
@@ -19,8 +19,8 @@ jobs:
       - name: Configure broken links checker
         run: |
           mkdir -p ./target
-          echo '{ "aliveStatusCodes": [429, 200] }' > ./target/broken_links_checker.json
-      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.13
+          echo '{ "aliveStatusCodes": [429, 200], "ignorePatterns": [{"pattern": "^https?://(www.)?opensource.org"}] }' > ./target/broken_links_checker.json
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           use-quiet-mode: 'yes'
           use-verbose-mode: 'yes'

--- a/dependencies.md
+++ b/dependencies.md
@@ -9,22 +9,22 @@
 | [Parquet for Java][2]                        | [MIT License][3]                                                                                             |
 | [error-reporting-java][4]                    | [MIT][1]                                                                                                     |
 | [Jakarta JSON Processing API][6]             | [Eclipse Public License 2.0][7]; [GNU General Public License, version 2 with the GNU Classpath Exception][8] |
-| [Performance Test Recorder Java][9]          | [MIT][1]                                                                                                     |
-| [Project Lombok][11]                         | [The MIT License][12]                                                                                        |
+| [Project Lombok][9]                          | [The MIT License][10]                                                                                        |
 
 ## Test Dependencies
 
 | Dependency                                   | License                           |
 | -------------------------------------------- | --------------------------------- |
-| [Hamcrest][13]                               | [BSD License 3][14]               |
-| [JUnit Jupiter Engine][15]                   | [Eclipse Public License v2.0][16] |
-| [JUnit Jupiter Params][15]                   | [Eclipse Public License v2.0][16] |
-| [mockito-core][19]                           | [The MIT License][20]             |
-| [mockito-junit-jupiter][19]                  | [The MIT License][20]             |
+| [Hamcrest][11]                               | [BSD License 3][12]               |
+| [JUnit Jupiter Engine][13]                   | [Eclipse Public License v2.0][14] |
+| [JUnit Jupiter Params][13]                   | [Eclipse Public License v2.0][14] |
+| [mockito-core][17]                           | [The MIT License][18]             |
+| [mockito-junit-jupiter][17]                  | [The MIT License][18]             |
 | [Common Virtual Schema for document data][0] | [MIT][1]                          |
-| [Matcher for SQL Result Sets][25]            | [MIT][1]                          |
-| [udf-debugging-java][27]                     | [MIT][1]                          |
-| [exasol-test-setup-abstraction-java][29]     | [MIT License][30]                 |
+| [Matcher for SQL Result Sets][23]            | [MIT][1]                          |
+| [udf-debugging-java][25]                     | [MIT][1]                          |
+| [exasol-test-setup-abstraction-java][27]     | [MIT License][28]                 |
+| [Performance Test Recorder Java][29]         | [MIT][1]                          |
 
 ## Runtime Dependencies
 
@@ -61,45 +61,45 @@
 
 [4]: https://github.com/exasol/error-reporting-java
 [41]: http://www.apache.org/licenses/LICENSE-2.0.txt
-[11]: https://projectlombok.org
+[9]: https://projectlombok.org
 [46]: https://maven.apache.org/surefire/maven-surefire-plugin/
 [70]: http://maven.apache.org/plugins/maven-clean-plugin/
 [1]: https://opensource.org/licenses/MIT
-[19]: https://github.com/mockito/mockito
+[17]: https://github.com/mockito/mockito
 [3]: https://github.com/exasol/parquet-io-java/blob/main/LICENSE
 [40]: https://www.mojohaus.org/flatten-maven-plugin/
 [48]: http://www.mojohaus.org/versions-maven-plugin/
 [56]: https://github.com/exasol/project-keeper/
-[14]: http://opensource.org/licenses/BSD-3-Clause
+[12]: http://opensource.org/licenses/BSD-3-Clause
 [36]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[29]: https://github.com/exasol/exasol-test-setup-abstraction-java/
+[27]: https://github.com/exasol/exasol-test-setup-abstraction-java/
 [67]: https://www.eclipse.org/legal/epl-2.0/
 [50]: https://maven.apache.org/plugins/maven-deploy-plugin/
 [35]: http://www.gnu.org/licenses/lgpl.txt
 [66]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
-[20]: https://github.com/mockito/mockito/blob/main/LICENSE
-[12]: https://projectlombok.org/LICENSE
-[25]: https://github.com/exasol/hamcrest-resultset-matcher
+[18]: https://github.com/mockito/mockito/blob/main/LICENSE
+[10]: https://projectlombok.org/LICENSE
+[23]: https://github.com/exasol/hamcrest-resultset-matcher
 [44]: http://zlika.github.io/reproducible-build-maven-plugin
-[30]: https://github.com/exasol/exasol-test-setup-abstraction-java/blob/main/LICENSE
+[28]: https://github.com/exasol/exasol-test-setup-abstraction-java/blob/main/LICENSE
 [34]: http://sonarsource.github.io/sonar-scanner-maven/
 [0]: https://github.com/exasol/virtual-schema-common-document/
-[27]: https://github.com/exasol/udf-debugging-java/
-[15]: https://junit.org/junit5/
+[25]: https://github.com/exasol/udf-debugging-java/
+[13]: https://junit.org/junit5/
 [6]: https://github.com/eclipse-ee4j/jsonp
 [54]: https://maven.apache.org/plugins/maven-source-plugin/
 [8]: https://projects.eclipse.org/license/secondary-gpl-2.0-cp
-[13]: http://hamcrest.org/JavaHamcrest/
+[11]: http://hamcrest.org/JavaHamcrest/
 [72]: http://maven.apache.org/plugins/maven-resources-plugin/
 [58]: https://maven.apache.org/plugins/maven-jar-plugin/
 [62]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
-[9]: https://github.com/exasol/performance-test-recorder-java
+[29]: https://github.com/exasol/performance-test-recorder-java
 [63]: http://www.eclipse.org/legal/epl-v10.html
 [57]: https://github.com/exasol/project-keeper/blob/main/LICENSE
 [7]: https://projects.eclipse.org/license/epl-2.0
 [37]: https://www.apache.org/licenses/LICENSE-2.0.txt
 [38]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
-[16]: https://www.eclipse.org/legal/epl-v20.html
+[14]: https://www.eclipse.org/legal/epl-v20.html
 [74]: http://maven.apache.org/plugins/maven-install-plugin/
 [42]: https://sonatype.github.io/ossindex-maven/maven-plugin/
 [52]: https://maven.apache.org/plugins/maven-gpg-plugin/

--- a/dependencies.md
+++ b/dependencies.md
@@ -6,7 +6,7 @@
 | Dependency                                   | License                                                                                                      |
 | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
 | [Common Virtual Schema for document data][0] | [MIT][1]                                                                                                     |
-| [Parquet for Java][2]                        | [MIT][1]                                                                                                     |
+| [Parquet for Java][2]                        | [MIT License][3]                                                                                             |
 | [error-reporting-java][4]                    | [MIT][1]                                                                                                     |
 | [Jakarta JSON Processing API][6]             | [Eclipse Public License 2.0][7]; [GNU General Public License, version 2 with the GNU Classpath Exception][8] |
 | [Performance Test Recorder Java][9]          | [MIT][1]                                                                                                     |
@@ -24,6 +24,7 @@
 | [Common Virtual Schema for document data][0] | [MIT][1]                          |
 | [Matcher for SQL Result Sets][25]            | [MIT][1]                          |
 | [udf-debugging-java][27]                     | [MIT][1]                          |
+| [exasol-test-setup-abstraction-java][29]     | [MIT License][30]                 |
 
 ## Runtime Dependencies
 
@@ -35,72 +36,75 @@
 
 | Dependency                                              | License                                        |
 | ------------------------------------------------------- | ---------------------------------------------- |
-| [SonarQube Scanner for Maven][32]                       | [GNU LGPL 3][33]                               |
-| [Apache Maven Compiler Plugin][34]                      | [Apache License, Version 2.0][35]              |
-| [Apache Maven Enforcer Plugin][36]                      | [Apache License, Version 2.0][35]              |
-| [Maven Flatten Plugin][38]                              | [Apache Software Licenese][39]                 |
-| [org.sonatype.ossindex.maven:ossindex-maven-plugin][40] | [ASL2][39]                                     |
-| [Reproducible Build Maven Plugin][42]                   | [Apache 2.0][39]                               |
-| [Maven Surefire Plugin][44]                             | [Apache License, Version 2.0][35]              |
-| [Versions Maven Plugin][46]                             | [Apache License, Version 2.0][35]              |
-| [Apache Maven Deploy Plugin][48]                        | [Apache License, Version 2.0][35]              |
-| [Apache Maven GPG Plugin][50]                           | [Apache License, Version 2.0][35]              |
-| [Apache Maven Source Plugin][52]                        | [Apache License, Version 2.0][35]              |
-| [Project keeper maven plugin][54]                       | [The MIT License][55]                          |
-| [Apache Maven JAR Plugin][56]                           | [Apache License, Version 2.0][35]              |
-| [Apache Maven Javadoc Plugin][58]                       | [Apache License, Version 2.0][35]              |
-| [Nexus Staging Maven Plugin][60]                        | [Eclipse Public License][61]                   |
-| [Lombok Maven Plugin][62]                               | [The MIT License][1]                           |
-| [JaCoCo :: Maven Plugin][64]                            | [Eclipse Public License 2.0][65]               |
-| [error-code-crawler-maven-plugin][66]                   | [MIT][1]                                       |
-| [Maven Clean Plugin][68]                                | [The Apache Software License, Version 2.0][39] |
-| [Maven Resources Plugin][70]                            | [The Apache Software License, Version 2.0][39] |
-| [Maven Install Plugin][72]                              | [The Apache Software License, Version 2.0][39] |
-| [Maven Site Plugin 3][74]                               | [The Apache Software License, Version 2.0][39] |
+| [SonarQube Scanner for Maven][34]                       | [GNU LGPL 3][35]                               |
+| [Apache Maven Compiler Plugin][36]                      | [Apache License, Version 2.0][37]              |
+| [Apache Maven Enforcer Plugin][38]                      | [Apache License, Version 2.0][37]              |
+| [Maven Flatten Plugin][40]                              | [Apache Software Licenese][41]                 |
+| [org.sonatype.ossindex.maven:ossindex-maven-plugin][42] | [ASL2][41]                                     |
+| [Reproducible Build Maven Plugin][44]                   | [Apache 2.0][41]                               |
+| [Maven Surefire Plugin][46]                             | [Apache License, Version 2.0][37]              |
+| [Versions Maven Plugin][48]                             | [Apache License, Version 2.0][37]              |
+| [Apache Maven Deploy Plugin][50]                        | [Apache License, Version 2.0][37]              |
+| [Apache Maven GPG Plugin][52]                           | [Apache License, Version 2.0][37]              |
+| [Apache Maven Source Plugin][54]                        | [Apache License, Version 2.0][37]              |
+| [Project keeper maven plugin][56]                       | [The MIT License][57]                          |
+| [Apache Maven JAR Plugin][58]                           | [Apache License, Version 2.0][37]              |
+| [Apache Maven Javadoc Plugin][60]                       | [Apache License, Version 2.0][37]              |
+| [Nexus Staging Maven Plugin][62]                        | [Eclipse Public License][63]                   |
+| [Lombok Maven Plugin][64]                               | [The MIT License][1]                           |
+| [JaCoCo :: Maven Plugin][66]                            | [Eclipse Public License 2.0][67]               |
+| [error-code-crawler-maven-plugin][68]                   | [MIT][1]                                       |
+| [Maven Clean Plugin][70]                                | [The Apache Software License, Version 2.0][41] |
+| [Maven Resources Plugin][72]                            | [The Apache Software License, Version 2.0][41] |
+| [Maven Install Plugin][74]                              | [The Apache Software License, Version 2.0][41] |
+| [Maven Site Plugin 3][76]                               | [The Apache Software License, Version 2.0][41] |
 
 [4]: https://github.com/exasol/error-reporting-java
-[2]: https://github.com/exasol/parquet-io-java
-[39]: http://www.apache.org/licenses/LICENSE-2.0.txt
+[41]: http://www.apache.org/licenses/LICENSE-2.0.txt
 [11]: https://projectlombok.org
-[44]: https://maven.apache.org/surefire/maven-surefire-plugin/
-[60]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
-[68]: http://maven.apache.org/plugins/maven-clean-plugin/
+[46]: https://maven.apache.org/surefire/maven-surefire-plugin/
+[70]: http://maven.apache.org/plugins/maven-clean-plugin/
 [1]: https://opensource.org/licenses/MIT
 [19]: https://github.com/mockito/mockito
-[9]: https://github.com/exasol/performance-test-recorder-java
-[46]: http://www.mojohaus.org/versions-maven-plugin/
-[54]: https://github.com/exasol/project-keeper/
+[3]: https://github.com/exasol/parquet-io-java/blob/main/LICENSE
+[40]: https://www.mojohaus.org/flatten-maven-plugin/
+[48]: http://www.mojohaus.org/versions-maven-plugin/
+[56]: https://github.com/exasol/project-keeper/
 [14]: http://opensource.org/licenses/BSD-3-Clause
-[34]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[65]: https://www.eclipse.org/legal/epl-2.0/
-[48]: https://maven.apache.org/plugins/maven-deploy-plugin/
-[61]: http://www.eclipse.org/legal/epl-v10.html
-[33]: http://www.gnu.org/licenses/lgpl.txt
-[64]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
+[36]: https://maven.apache.org/plugins/maven-compiler-plugin/
+[29]: https://github.com/exasol/exasol-test-setup-abstraction-java/
+[67]: https://www.eclipse.org/legal/epl-2.0/
+[50]: https://maven.apache.org/plugins/maven-deploy-plugin/
+[35]: http://www.gnu.org/licenses/lgpl.txt
+[66]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
 [20]: https://github.com/mockito/mockito/blob/main/LICENSE
 [12]: https://projectlombok.org/LICENSE
 [25]: https://github.com/exasol/hamcrest-resultset-matcher
-[42]: http://zlika.github.io/reproducible-build-maven-plugin
-[55]: https://github.com/exasol/project-keeper/blob/main/LICENSE
-[7]: https://projects.eclipse.org/license/epl-2.0
-[32]: http://sonarsource.github.io/sonar-scanner-maven/
-[35]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[36]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
-[62]: https://awhitford.github.com/lombok.maven/lombok-maven-plugin/
+[44]: http://zlika.github.io/reproducible-build-maven-plugin
+[30]: https://github.com/exasol/exasol-test-setup-abstraction-java/blob/main/LICENSE
+[34]: http://sonarsource.github.io/sonar-scanner-maven/
 [0]: https://github.com/exasol/virtual-schema-common-document/
 [27]: https://github.com/exasol/udf-debugging-java/
-[16]: https://www.eclipse.org/legal/epl-v20.html
-[72]: http://maven.apache.org/plugins/maven-install-plugin/
 [15]: https://junit.org/junit5/
-[40]: https://sonatype.github.io/ossindex-maven/maven-plugin/
-[50]: https://maven.apache.org/plugins/maven-gpg-plugin/
-[38]: https://www.mojohaus.org/flatten-maven-plugin/flatten-maven-plugin
 [6]: https://github.com/eclipse-ee4j/jsonp
-[52]: https://maven.apache.org/plugins/maven-source-plugin/
+[54]: https://maven.apache.org/plugins/maven-source-plugin/
 [8]: https://projects.eclipse.org/license/secondary-gpl-2.0-cp
 [13]: http://hamcrest.org/JavaHamcrest/
-[74]: http://maven.apache.org/plugins/maven-site-plugin/
-[70]: http://maven.apache.org/plugins/maven-resources-plugin/
-[58]: https://maven.apache.org/plugins/maven-javadoc-plugin/
-[66]: https://github.com/exasol/error-code-crawler-maven-plugin
-[56]: https://maven.apache.org/plugins/maven-jar-plugin/
+[72]: http://maven.apache.org/plugins/maven-resources-plugin/
+[58]: https://maven.apache.org/plugins/maven-jar-plugin/
+[62]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
+[9]: https://github.com/exasol/performance-test-recorder-java
+[63]: http://www.eclipse.org/legal/epl-v10.html
+[57]: https://github.com/exasol/project-keeper/blob/main/LICENSE
+[7]: https://projects.eclipse.org/license/epl-2.0
+[37]: https://www.apache.org/licenses/LICENSE-2.0.txt
+[38]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
+[16]: https://www.eclipse.org/legal/epl-v20.html
+[74]: http://maven.apache.org/plugins/maven-install-plugin/
+[42]: https://sonatype.github.io/ossindex-maven/maven-plugin/
+[52]: https://maven.apache.org/plugins/maven-gpg-plugin/
+[2]: https://github.com/exasol/parquet-io-java/
+[64]: https://anthonywhitford.com/lombok.maven/lombok-maven-plugin/
+[76]: http://maven.apache.org/plugins/maven-site-plugin/
+[60]: https://maven.apache.org/plugins/maven-javadoc-plugin/
+[68]: https://github.com/exasol/error-code-crawler-maven-plugin

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,5 +1,6 @@
 # Changes
 
+* [7.0.2](changes_7.0.2.md)
 * [7.0.1](changes_7.0.1.md)
 * [7.0.0](changes_7.0.0.md)
 * [6.0.2](changes_6.0.2.md)

--- a/doc/changes/changes_6.0.2.md
+++ b/doc/changes/changes_6.0.2.md
@@ -4,7 +4,7 @@ Code name: 6.0.2: Upgrade dependencies compare to 6.0.1
 
 ## Features
 
-* #98: Upgraded dependencies to fix [CVE-2022-21724](https://ossindex.sonatype.org/vulnerability/0f319d1b-e964-4471-bded-db3aeb3c3a29?component-type=maven&component-name=org.postgresql.postgresql&utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1) in the PostgreSQL JDBC driver.
+* #98: Upgraded dependencies to fix [CVE-2022-21724](https://nvd.nist.gov/vuln/detail/CVE-2022-21724) in the PostgreSQL JDBC driver.
 
 ## Dependency Updates
 

--- a/doc/changes/changes_7.0.2.md
+++ b/doc/changes/changes_7.0.2.md
@@ -15,10 +15,12 @@ This release fixes vulnerabilities in dependencies by updating compile dependenc
 ### Compile Dependency Updates
 
 * Updated `com.exasol:parquet-io-java:1.3.0` to `1.3.1`
+* Removed `com.exasol:performance-test-recorder-java:0.1.0`
 
 ### Test Dependency Updates
 
 * Added `com.exasol:exasol-test-setup-abstraction-java:0.3.2`
+* Added `com.exasol:performance-test-recorder-java:0.1.0`
 * Updated `org.mockito:mockito-core:4.5.1` to `4.6.1`
 * Updated `org.mockito:mockito-junit-jupiter:4.5.1` to `4.6.1`
 

--- a/doc/changes/changes_7.0.2.md
+++ b/doc/changes/changes_7.0.2.md
@@ -1,0 +1,29 @@
+# Virtual Schema for document data in files 7.0.2, released 2022-06-09
+
+Code name: 7.0.2: Upgrade dependencies on 7.0.1
+
+## Summary
+
+This release fixes vulnerabilities in dependencies by updating compile dependency.
+
+## Features
+
+* #104: Upgraded dependencies to fix vulnerabilities
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `com.exasol:parquet-io-java:1.3.0` to `1.3.1`
+
+### Test Dependency Updates
+
+* Added `com.exasol:exasol-test-setup-abstraction-java:0.3.2`
+* Updated `org.mockito:mockito-core:4.5.1` to `4.6.1`
+* Updated `org.mockito:mockito-junit-jupiter:4.5.1` to `4.6.1`
+
+### Plugin Dependency Updates
+
+* Updated `com.exasol:project-keeper-maven-plugin:2.3.2` to `2.4.6`
+* Updated `org.codehaus.mojo:versions-maven-plugin:2.10.0` to `2.11.0`
+* Updated `org.jacoco:jacoco-maven-plugin:0.8.7` to `0.8.8`

--- a/pk_generated_parent.pom
+++ b/pk_generated_parent.pom
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-document-files-generated-parent</artifactId>
-    <version>7.0.1</version>
+    <version>7.0.2</version>
     <packaging>pom</packaging>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.20</version>
+            <version>1.18.24</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
@@ -50,7 +50,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.9.0</version>
+                <version>3.10.1</version>
                 <configuration>
                     <source>${java.version}</source>
                     <target>${java.version}</target>
@@ -104,7 +104,7 @@
             <plugin>
                 <groupId>org.sonatype.ossindex.maven</groupId>
                 <artifactId>ossindex-maven-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>3.2.0</version>
                 <executions>
                     <execution>
                         <id>audit</id>
@@ -143,7 +143,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>versions-maven-plugin</artifactId>
-                <version>2.8.1</version>
+                <version>2.10.0</version>
                 <executions>
                     <execution>
                         <id>display-updates</id>
@@ -202,7 +202,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.3.1</version>
+                <version>3.4.0</version>
                 <executions>
                     <execution>
                         <id>attach-javadocs</id>
@@ -222,7 +222,7 @@
             <plugin>
                 <groupId>org.sonatype.plugins</groupId>
                 <artifactId>nexus-staging-maven-plugin</artifactId>
-                <version>1.6.8</version>
+                <version>1.6.13</version>
                 <configuration>
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                     <serverId>ossrh</serverId>
@@ -272,7 +272,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.7</version>
+                <version>0.8.8</version>
                 <executions>
                     <execution>
                         <id>prepare-agent</id>
@@ -313,7 +313,7 @@
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>error-code-crawler-maven-plugin</artifactId>
-                <version>1.1.0</version>
+                <version>1.1.1</version>
                 <executions>
                     <execution>
                         <id>verify</id>

--- a/pom.xml
+++ b/pom.xml
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-document-files</artifactId>
-    <version>7.0.1</version>
+    <version>7.0.2</version>
     <name>Virtual Schema for document data in files</name>
     <description>Adapter for document data access from files.</description>
     <url>https://github.com/exasol/virtual-schema-common-document-files/</url>
@@ -40,7 +39,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>parquet-io-java</artifactId>
-            <version>1.3.0</version>
+            <version>1.3.1</version>
         </dependency>
         <!-- Test dependencies -->
         <dependency>
@@ -69,13 +68,13 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>4.5.1</version>
+            <version>4.6.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-junit-jupiter</artifactId>
-            <version>4.5.1</version>
+            <version>4.6.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
@@ -112,6 +111,13 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <!-- Explicitly upgrade dependency to fix vulnerabilities -->
+            <groupId>com.exasol</groupId>
+            <artifactId>exasol-test-setup-abstraction-java</artifactId>
+            <version>0.3.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>performance-test-recorder-java</artifactId>
             <version>0.1.0</version>
@@ -138,12 +144,23 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>versions-maven-plugin</artifactId>
-                <version>2.10.0</version>
+                <version>2.11.0</version>
             </plugin>
             <plugin>
                 <groupId>org.sonatype.ossindex.maven</groupId>
                 <artifactId>ossindex-maven-plugin</artifactId>
                 <version>3.2.0</version>
+                <configuration>
+                    <excludeVulnerabilityIds>
+                        <!-- com.google.guava:guava: https://ossindex.sonatype.org/vulnerability/sonatype-2020-0926
+                        Excluded because we assume that the attacker does not have access to the machine. -->
+                        <exclude>sonatype-2020-0926</exclude>
+                        <!-- io.netty:netty-handler: https://ossindex.sonatype.org/vulnerability/sonatype-2020-0026
+                        Not a vulnerability in netty but how it is configured by the client software.amazon.awssdk:netty-nio-client
+                        which is only used in tests. -->
+                        <exclude>sonatype-2020-0026</exclude>
+                    </excludeVulnerabilityIds>
+                </configuration>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
@@ -153,7 +170,7 @@
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>project-keeper-maven-plugin</artifactId>
-                <version>2.3.2</version>
+                <version>2.4.6</version>
                 <executions>
                     <execution>
                         <goals>
@@ -225,7 +242,7 @@
     <parent>
         <artifactId>virtual-schema-common-document-files-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>7.0.1</version>
+        <version>7.0.2</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -121,12 +121,7 @@
             <groupId>com.exasol</groupId>
             <artifactId>performance-test-recorder-java</artifactId>
             <version>0.1.0</version>
-        </dependency>
-        <dependency>
-            <artifactId>lombok</artifactId>
-            <groupId>org.projectlombok</groupId>
-            <version>1.18.24</version>
-            <scope>provided</scope>
+            <scope>test</scope>
         </dependency>
     </dependencies>
     <build>

